dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESReadOnlyMonitor.java

```diff
@@ -0,0 +1,145 @@
+package com.dotcms.content.elasticsearch.business;
+
+import com.dotcms.api.system.event.message.MessageSeverity;
+import com.dotcms.api.system.event.message.MessageType;
+import com.dotcms.api.system.event.message.SystemMessageEventUtil;
+import com.dotcms.api.system.event.message.builder.SystemMessageBuilder;
+import com.dotcms.concurrent.DotConcurrentFactory;
+import com.dotcms.enterprise.LicenseUtil;
+import com.dotcms.repackage.com.google.common.annotations.VisibleForTesting;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.business.Role;
+import com.dotmarketing.business.RoleAPI;
+import com.dotmarketing.common.reindex.ReindexEntry;
+import com.dotmarketing.common.reindex.ReindexThread;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.util.Logger;
+import com.liferay.portal.language.LanguageException;
+import com.liferay.portal.language.LanguageUtil;
+
+import java.util.List;
+import java.util.Timer;
+import java.util.TimerTask;
+import java.util.concurrent.ScheduledThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.stream.Collectors;
+
+import static com.dotcms.util.CollectionsUtils.list;
+
+/**
+ * Run a Monitor If the current indices are in read only mode and try to set them to write mode again.
+ * When the set write mode fail it try again after one minute.
+ */
+public class ESReadOnlyMonitor {
```

Comment:
Please correct by using text from below:
```
/**
 * This monitor is run when either the Live or Working indices are in read-only mode.  It attempts to set both Live and Working indices to write mode again.
 * When setting write-mode fails it will retry after one minute.
 */
```

Reply:
done https://github.com/dotCMS/core/pull/18604/commits/fc979e0e1d04475951dbb213b379b17022474a8e#diff-8528ec811f60c9aae6301f9071d59872R27